model/postgres/postgres_resource_firewall_rule.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require_relative "../../model"
+
+class PostgresResourceFirewallRule < Sequel::Model
```

Comment:
I think PostgresFirewallRule is enough. It also fits nicely with other Postgres related models.